webui/react/src/ioTypes.ts

```diff
@@ -21,6 +21,8 @@ export const decode = <T>(type: io.Mixed, data: any): T => {
   }
 };
 
+const ioNullUndefined = io.union([ io.null, io.undefined ]);
```

Comment:
nice, this is great

Reply:
should save us from some surprises.

Reply:
I was gonna go ahead and update ioTypes to use this in more places but to avoid more conflicts and delaying this work I'll leave that for another PR

Reply:
sounds great